recipes/wasmtime/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.1)
+project(PackageTest)
+
+set(C_STANDARD 11)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+find_package(wasmtime REQUIRED)
+
+add_executable(example example.c)
+target_link_libraries(example PRIVATE wasmtime::wasmtime)
```

Don't use `find_package`. wasmtime does not provide cmake scripts in their binaries.
```suggestion
conan_basic_setup()

add_executable(example example.c)
target_link_libraries(example PRIVATE ${CONAN_LIBS})
```

---

Previously I was suggested not to use `CONAN_LIBS`

---

Are you sure?
That would be contradictory to all recipes we currently have in our repo.
We only test cmake modules (e.g. `wasmtime-config.cmake`) when the upstream project builds/ships them.
Or if cmake has a `FindXXX.cmake` module.

---

In `cmake-conan` project I was suggested not to use this feature because it will be deprecated, instead was suggested to use alias target instead

---

Indeed, the `cmake` generator (that creates `conanbuildinfo.cmake` will be deprecated in conan 2.0).
You're also using it here, so we'll have to update this test recipe + cmake script anyways.

---

But what if I want to test using this package by specific generator ...
For me it is subjective changes

---

Done